conftest.py

```diff
@@ -13,3 +16,25 @@ def pytest_funcarg__mozwebqa(request):
         lambda s: s.find_element_by_css_selector(fxa_login_button_locator_css).is_displayed())
     mozwebqa.selenium.find_element_by_css_selector(fxa_login_button_locator_css).click()
     return mozwebqa
+
+
+@pytest.fixture
+def dev_account(request):
```

Comment:
Thanks. I changed that, although we still need a `dev_account` fixture as we don't need or want all of the tests running against prod.